### PR TITLE
feat: add api to get mau summaries

### PR DESCRIPTION
**Pull request by masaaania**
- This PR
  - adds an API to summarize MAU. https://github.com/bucketeer-io/bucketeer/pull/554/commits/78f10e0f02a01405d124fd44aabfd0267b8ce194
  - adds a cronjob to execute that API. https://github.com/bucketeer-io/bucketeer/pull/554/commits/6094e9d406563eefa04da89ae82bd1b9b15d2aa7

### CODE REVIEW

File: proto/eventcounter/service.proto [hunk 1]
```diff
@@ -130,6 +131,15 @@ message GetMAUCountResponse {
   int64 user_count = 2;
 }
 
+message GetMAUCountsRequest {
+  string year_month = 1;
+  bool isGroupingBySourceID = 2;
```
Comment (cre8ivejp) [inline proto/eventcounter/service.proto:136]: nit: `is_grouping_by_source_id`

File: pkg/eventcounter/storage/v2/user_count.go [hunk 2]
```diff
@@ -65,3 +75,98 @@ func (s *userCountStorage) GetMAUCount(
...
+	ctx context.Context,
+	yearMonth string,
+) ([]*proto.MAUSummary, error) {
+	summaries := make([]*proto.MAUSummary, 0)
+	query := `
+		SELECT
+			environment_namespace as environment_id,
+			source_id,
+			count(*) as user_count,
+			IFNULL(SUM(event_count), 0) as request_count
```
Comment (cre8ivejp) [inline pkg/eventcounter/storage/v2/user_count.go:135]: How about adding the evaluation and goal request count to the query?

File: pkg/eventcounter/api/api_test.go [hunk 3]
```diff
@@ -786,6 +786,106 @@ func TestGetMAUCount(t *testing.T) {
 	}
 }
 
+func TestSummarizeMAUCounts(t *testing.T) {
```
Comment (cre8ivejp) [inline pkg/eventcounter/api/api_test.go:789]: Can you also add the tests for `Unauthenticated`, `PermissionDenied`, and `Internal` while calling the `checkAdminRole`?

Comment (masaaania) [reply pkg/eventcounter/api/api_test.go:789]: @cre8ivejp 
Thanks, I added it 👍

**Review by cre8ivejp — APPROVED** [review]
Thank you 🎉

**Pull request merged**